app/src/main/java/com/example/tasky/agenda/domain/util/AgendaItemType.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.tasky.agenda.domain.util
+
+enum class AgendaItemType {
+    TASK,
+    EVENT,
+    REMINDER
+}
+
+fun AgendaItemType.toInt(): Int = when (this) {
+    AgendaItemType.TASK -> 0
```

Comment:
I believe with [this approach](https://github.com/ArturGubala/Tasky/pull/21/files#r2380725194)  this extra mappers  can be avoided  and take advantage of the proper methods and properties of the enum class such as:

```kotlin
// Enum to String
val typeString: String = AgendaItemType.TASK.name  // "TASK"

// String back to Enum
val type: AgendaItemType = AgendaItemType.valueOf(typeString) // AgendaItemType.TASK
```